app/services/chat_notifications/providers/slack.rb

```diff
@@ -0,0 +1,19 @@
+class ChatNotifications::Providers::Slack
+  WEBHOOK_URL = ::BillingConfig['notifications']['webhook_url']
+  CHANNEL = ::BillingConfig['notifications']['room']
```

Comment:
any of them are mandatory but if we don't specify the username the notifications will be sent as "Incoming webhook"